app/src/main/res/values/strings.xml

```diff
@@ -52,4 +53,8 @@
     <string name="get">GET</string>
     <string name="retrofit">Retrofit</string>
     <string name="retrofit_and_coroutine">Retrofit and Coroutine</string>
+    <string name="layout">Layout</string>
+    <string name="component">Component</string>
+    <string name="this_is_a_title">Guideline is very useful</string>
+    <string name="sample_body">This is a body of something. This is very loooooooooooooooooooooooooooooooooooooong string. \n\n Actually this is really looong string.\n\nThen this is a verrrrrrrrrrrrrrrrry long text. Because it's needed truly.</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/punchdrunker/hocho/blob/24bb0f2642dd5edd85f0d053278a799e8a33704a/app/src/main/res/values/strings.xml#L59"></span>Apostrophe not preceded by \</td>
    </tr>
  </tbody>
</table>